### PR TITLE
Remove 3.3 from testing and building.

### PR DESCRIPTION
**Pull request by damianavila**
According to https://github.com/bokeh/bokeh/pull/2743#issuecomment-132731963

### CODE REVIEW

File: scripts/build_upload.sh [hunk 1]
```diff
@@ -96,7 +96,7 @@ echo "Removing first bokeh build at $first_build_loc"
 #########################
 
 # build for each python version
-for py in 27 33 34;
```
Comment (bryevdv) [inline scripts/build_upload.sh]: Will conda install the 3.4 version for 3.3 users?

Comment (damianavila) [inline scripts/build_upload.sh]: I don't think so... but if you want to use the 3.4 conda package you just need to make a 3.4 environment... I don't think it is too much work to ask...

Comment (bryevdv) [inline scripts/build_upload.sh]: Thinking about people that might have to use 3.3 without a choice (organizational mandate, etc)

Comment (damianavila) [inline scripts/build_upload.sh]: well... we can still provide the py33 packages... but if we are not testing in py33, we could be shipping a non working thing... I think we need to be consistent...

**Comment by bryevdv** [comment]
@pzwang do you have any thoughts or concerns?

**Comment by bryevdv** [comment]
@bokeh/dev  according to this (not sure about how reliable) in 2015Q1 py33 accounted for 12% of usage:

http://blog.jelastic.com/2015/04/23/software-stacks-market-share-first-quarter-2015/

Maybe too early to drop support.

**Comment by bryevdv** [comment]
Counterargument: could be measuring automated (e.g. TravisCI) usage in those numbers.

**Comment by bryevdv** [comment]
After some FD and other discussions I guess I am +1 on this after all. @damianavila ?

**Comment by damianavila** [comment]
:+1: as well... this is way I propose it :wink:

**Comment by bryevdv** [comment]
Ok, merging.

**Comment by damianavila** [comment]
grr.... this was merged against the dev_0.10 branch and never got merged on master... do you want me to reopen the PR against master and merge it?

**Comment by damianavila** [comment]
We also should remove the dev_0.10 to prevent this happening again...

**Comment by bryevdv** [comment]
@damianavila sorry this is on me, not used to checking for PRs against branches other than master. Sure, make a new PR and I will delete `dev_0.10` now.

**Comment by damianavila** [comment]
Reopened here: https://github.com/bokeh/bokeh/pull/2743#event-408214308

**Pull request merged**